website/docs/r/cloudaccount_aws.html.markdown

```diff
@@ -111,6 +111,31 @@ resource "dome9_cloudaccount_AWS" "test" {
 }
 ```
 
+```hcl
+resource "dome9_cloudaccount_awsgov" "test" {
+  name  = "ACCOUNT NAME"
+ 
```

Comment:
add vendor property to awsgov

Reply:
U right... it's always dome9_cloudaccount_aws.
my bad.